src/main/java/Storage.java

```diff
@@ -0,0 +1,149 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Storage class to represent the tasks being stored into hard drive.
+ * Contains file location of saved task and the file f representing the saved tasks.
+ */
+public class Storage {
+
+    private String fileLocation = "./Data/Tasks.txt"; //hard-coded relative file location of stored tasks
+    private File file;
+
+    // initialization of Storage
+
+    /**
+     * Constructor for the storage class.
+     */
+    public Storage() {
+        this.file = new File(this.fileLocation);
+    }
+
+    /**
+     * Function to check if the file exists or not.
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.). Instead of starting with "Function...", you could start with the description of what the method is doing (i.e. "Checks if..."). There should also be an empty line between description and parameter section.

This applies to all methods in this class.